interface/patient_file/encounter/superbill_custom_full.php

```diff
@@ -11,20 +11,31 @@
 require_once("../../../custom/code_types.inc.php");
 require_once("$srcdir/options.inc.php");
 
+// gacl control
+$thisauthview = acl_check('admin', 'superbill', false, 'view');
+$thisauthwrite = acl_check('admin', 'superbill', false, 'write');
+
+if (!($thisauthwrite || $thisauthview)) {
+    echo "<html>\n<body>\n";
+    echo "<p>".xl('You are not authorized for this.','','','</p>')."\n";
```

Comment:
Do this instead for above line:
``````
echo "<p>" . xlt('You are not authorized for this.') . "</p>\n";